source/core/TypeAndDataManager.cpp

```diff
@@ -2876,6 +2894,22 @@ VIREO_FUNCTION_SIGNATURE4(TypeGetSubElementFromPath, StaticType, void, StringRef
   //  tm->PointerToSymbolPath(_Param(1), _Param(2), _Param(3));
     return _NextInstruction();
 }
+//------------------------------------------------------------
+//! Parse through a path from a starting point to target subelement, return subelement value if it matches expected output type.
+VIREO_FUNCTION_SIGNATURE5(GetSubElementFromPath, StaticType, void, StringRef, StaticType, void)
```

Comment:
Optional parameters only make sense when there is a well-defined behavior without them.
The first two parameters are required to function, and if the only output is optional, the whole thing is a no-op.